src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -934,7 +934,7 @@ public void It_can_build_with_dynamic_loading_enabled(string targetFramework, st
             var outputDirectory = buildCommand.GetOutputDirectory(testProject.TargetFrameworks);
             outputDirectory.Should().HaveFiles(new[] {
                 runtimeConfigName,
-                $"{testProject.Name}.runtimeconfig.dev.json"
```

Comment:
was this file renamed from 3.0 to now?

Reply:
No, we stopped generating it: https://github.com/dotnet/sdk/pull/17014